docs/examples/resources.rst

```diff
@@ -160,6 +160,26 @@ Any records not passing validation will cause the entire operation to fail.
     )
 
 
+Batch Record Delete
+^^^^^^^^^^^^^^^^^^^
+Delete multiple records at once.
+
+.. code-block:: python
+
+    # Delete by record
+    records = app.records.search(limit=0) # list all records
```

Comment:
Update to manually grab three records, and show it being called with all the records.